scripts/systemd/falcoctl.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Falcoctl: Administrative tooling for Falco
+Documentation=https://github.com/falcosecurity/falcoctl
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/falcoctl --follow
```

Comment:
Is this the correct command? @alacuku 

Permissions on the unit should also be reviewed and then tweaked/fixed.